run.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+LROOT=$PWD
+ROOTFS_X86=_install_x86
+ROOTFS_ARM=_install_arm
```

Comment:
should be _install_arm32

Reply:
As I comment above on the patch.
Currently only support x86 platform. I will support that later